### ISSUE TITLE
audio silent when encode in MP4 with ffmpeg since version 2.0.0

### ISSUE THREAD

**Issue by ROBERT-MCDOWELL** [mstorsjo/fdk-aac#95]
I use libfdk_aac with ffmpeg since years now, but
suddenly since one month (maybe linked to version 2.0.0?)
all encoding done with ffmpeg nothing is audible, although the code is ok on the file info.
any idea of what's happening?

**Comment by mstorsjo** [comment]
I presume you rebuilt ffmpeg also after updating fdk-aac? The new version, 2.0.0, is not ABI compatible with the older ones.

If you still have this issue after rebuilding ffmpeg, can you provide a specific sample of commands for encoding (possibly with a sample input file), where you can reproduce the issue? Because for me, it seems to work fine in the normal use cases at least.

**Comment by ROBERT-MCDOWELL** [comment]
Usually I use a very simple command like
ffmpeg -i xxx -c:a libfdk_aac  etc....
nothing special.
I compiled fdk-aac last git like
./configure --prefix=/usr --libdir=/usr/lib64

compiled ffmpeg like
./configure --enable-gpl --enable-nonfree --enable-libx264 --enable-libspeex --enable-librtmp --enable-libfreetype --enable-libxcb --enable-libxcb-shm --enable-libxcb-xfixes --enable-libxcb-shape --enable-libfdk-aac --enable-libopenh264 --enable-shared --enable-libcaca --enable-libxcb --enable-libxcb-shm --enable-libxcb-xfixes --enable-libxcb-shape --enable-libv4l2 --enable-version3

most of the sample I use to encode come from f4v/flv h264/speex

**Comment by ROBERT-MCDOWELL** [comment]
btw I removed all 2.0.0 files, reinstalled the fdk-aac Fedora 27 package
and everything is working back again.

**Comment by mstorsjo** [comment]
Sounds to me like there might have been some compile/link time mixup, with including headers from one version of fdk-aac but actually linking against another version.

**Comment by ROBERT-MCDOWELL** [comment]
for the 2.0.0 tests I completely removed any traces of version 1, including
the packages and updated with ldconfig and libtool

**Comment by ROBERT-MCDOWELL** [comment]
where can I find the fdk-aac headers if any?

**Comment by ROBERT-MCDOWELL** [comment]
good guess, it was the headers...
but it seems that version 2 does not compile anymore with libav-11.xx

**Comment by mstorsjo** [comment]
Yes, version 2 broke the api and is only supported by the latest git versions of libav/ffmpeg.
